doc/string/bytes.rdoc

```diff
@@ -0,0 +1,9 @@
+Returns an array of the bytes in +self+:
+
+  'hello'.bytes # => [104, 101, 108, 108, 111]
+  'тест'.bytes  # => [209, 130, 208, 181, 209, 129, 209, 130]
+  'こんにちは'.bytes
+  # => [227, 129, 147, 227, 130, 147, 227, 129, 171, 227, 129, 161, 227, 129, 175]
+
+
+
```

Comment:
Can you remove the extra newlines at the end of the files?

Reply:
Done.